cmd/api/database.go

```diff
@@ -88,16 +88,18 @@ func searchById(id uuid.UUID) (Book, error) {
 }
 
 /* Returns filtered content of database in a list of books*/
-func listBooks(name string, minPrice32, maxPrice32 float32) ([]Book, error) {
+func listBooks(name string, minPrice32, maxPrice32 float32, sortBy, sortDirection string) ([]Book, error) {
 	if name == "" {
 		name = "%"
 	}
+	name = fmt.Sprint("%", name, "%")
+	fmt.Println(name)
```

Comment:
Is this debug code?
If it is, please remove it.

Reply:
Yes. Sorry